extensions/extended-blocks/paid-blocks/paid-blocks.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Block Editor functionality for Paid Blocks.
+ *
+ * @package Automattic\Jetpack\Extended_Blocks
+ */
+
+// RequirePaid_Blocks class.
+require_once './class-paid-blocks.php';
+Automattic\Jetpack\Extended_Blocks\Paid_Blocks::get_instance();
```

Comment:
```suggestion
// RequirePaid_Blocks class.
require_once './class-paid-blocks.php';

Automattic\Jetpack\Extended_Blocks\Paid_Blocks::get_instance();
$instance = Automattic\Jetpack\Extended_Blocks\Paid_Blocks::get_instance();
$instance->init(\Jetpack_Gutenberg);
```

This would allow us to inject the dependency which would allow to easier unit testing of the code.

Obviously you'd need to update the `Paid_Blocks` class to add the `init()` method and to accept the `Jetpack_Gutenberg` class dependency.

Reply:
Reverting this commit. Let's keep it as simple as possible. We can iterate later.

Reply:
> We can iterate later.

🤞 😉